sdk/app_cpu1/common/sys/log.c

```diff
@@ -488,6 +505,10 @@ void state_machine_dump_binary_callback(void *arg)
 
     case DUMP_BINARY_SAMPLE_INTERVAL_USEC:
     {
+        // WARNING: A control frequency that results in a non-integer time interval period (example: 22 kHz / 45.45 us)
+        //          will have that interval truncated (example, to 45 us) here. The host PC will have an incorrect time
+        //          vector for any logged data in this case!!
+        //          See https://docs.amdc.dev/getting-started/user-guide/logging/index.html
```

Comment:
Can you remove the `index.html` from this link? It will still work and it looks cleaner ;)